Dockerfile

```diff
@@ -11,7 +11,6 @@ RUN make build
 
 FROM alpine:latest
 RUN apk add --no-cache ca-certificates
-RUN apk add openssl=3.1.4-r6 && apk upgrade openssl --no-cache
```

Comment:
Just noticed this; why no need for openssl anymore? Or was it there by accident in the first place?